src/main/twirl/uk/gov/hmrc/hmrcfrontend/views/helpers/HmrcPageHeadingHelper.scala.html

```diff
@@ -0,0 +1,29 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.hmrcfrontend.views.html.components.HmrcPageHeading
+@import uk.gov.hmrc.hmrcfrontend.views.viewmodels.pageheading.PageHeading
+@import uk.gov.hmrc.hmrcfrontend.views.viewmodels.pageheading.PageHeadingHelper
+
+@this(hmrcNewTabLink: HmrcPageHeading)
+
+
+@(pageHeadingHelper: PageHeadingHelper)(implicit messages: Messages)
+@hmrcNewTabLink(PageHeading(
+  text = pageHeadingHelper.text,
+  section = pageHeadingHelper.section,
+  language = Some(messages.lang.language)
+))
```

Comment:
@kyle-bowden @JoPintoPaul it feels like we are adding a lot of extra stuff to the API for developers for this, and I wonder if we should consider an implicit or a helper for constructing just the viewmodel, so for example something like one of these

```scala
hmrcPageHeading(LocalisedPageHeading(text="page title"))
hmrcPageHeading(PageHeading(text="page title").withLanguage)
```

Reply:
It makes sense to me to actually go this way instead of adding an additional helper component.

I like the bellow approach 
`hmrcPageHeading(PageHeading(text="page title").withLanguage)`

Reply:
I agree, I think with an implicit like that you get some discoverability of the API via IDE completions, where as you have to know about the helpers and remember the when you need them. I guess also it doesn't actually need to be an implicit, pretty small change then too, don't really even need an implicit (like below), but could do it like that still for consistency

```
case class PageHeading(title: String, language: String) {
  def withLanguage(implicit lang: Lang): PageHeading = this.copy(
    language = lang.code match {
      case "cy" => "cy"
      case _    => "en"
    }
  )
}
```

Reply:
(that code is total psuedocode, can't remember how to get language code out and I know we have En / Cy classes not strings)